Genfare/Utility/GFUtilities.swift

```diff
@@ -85,6 +86,45 @@ class Utilities {
     class func accountBalance() -> NSNumber {
         return UserDefaults.standard.value(forKey: Constants.LocalStorage.AccountBalance) as! NSNumber
     }
+    class func walletContentsBalance() -> NSNumber{
+        let products = GFFetchProductsService.getProducts()
+        let items = products.filter({ (product:Product) -> Bool in
+             product.ticketTypeDescription == "Stored Value" && product.isActivationOnly == 0
+        })
+        if items.count > 0{
+            if let prod = items[0] as? Product{
+                if let ticketId = prod.ticketId?.intValue{
+                    do{
+                        var userObj:WalletContents
+                        let managedContext = GFDataService.context
+                        let fetchRequest:NSFetchRequest = WalletContents.fetchRequest()
+                        let strTicketId = String(format: "%d", ticketId)
+                        fetchRequest.predicate = NSPredicate(format: "ticketIdentifier == %@",strTicketId)
+                        let fetchResults = try managedContext.fetch(fetchRequest)
+                        if fetchResults.count >= 0 {
+                            guard let firstObj = fetchResults.first else{ return NSNumber.init(value: Float(0)) }
+                            userObj = firstObj
+                            guard let balance = userObj.balance else { return NSNumber.init(value: Float(0)) }
+                            return NSNumber.init(value: Float(balance)!)
+                        }
+                    }catch{
+                        print("saving failed ")
+                    }
+                    return NSNumber.init(value: Float(0))
```

Comment:
You can return simply 0 like this, I think you done need to use NSNumber.init() 
return 0

Reply:
Done.